### DISCUSSION TITLE
API Quota

### DISCUSSION THREAD

**Discussion by simpsss** [AzuraCast/AzuraCast#4020] [Q&A]
Hello,

I'm going back to using Azuracast and I have a question, before to set the quota in the API, I used "storage_quota". How do I do it now?

Thanks

**Comment by Vaalyn** [comment]
The `storageQuota` is now part of the `Storage Locations` API endpoint that defines the storage locations used for media, backups and recordings. See the `Administration: Storage Locations` section of the Swagger API documentation: https://demo.azuracast.com/static/api/index.html#/Administration%3A%20Storage%20Locations

**Comment by simpsss** [comment]
is it possible to set the quota via API /admin/stations ?

**Comment by Vaalyn** [comment]
Nope, the quota is not tied to the `Station` but to the `Storage Location` entity so you need to use the API endpoints for that entity.
